orttraining/orttraining/python/ort_trainer.py

```diff
@@ -11,6 +11,7 @@
 import onnxruntime as ort
 from distutils.version import LooseVersion
 from .checkpointing_utils import list_checkpoint_files, get_checkpoint_name, CombineZeroCheckpoint
+import onnxruntime.capi.pt_patch
```

Comment:
Seems it is not being used

Reply:
just importing pt_patch will patch the nll_loss for pytorch.